test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -1402,7 +1471,7 @@ static public void testStaticFieldsWithObjectAlignment() throws Throwable {
 	static public void testCreateLargeNumberOfPoint2D() throws Throwable {
 		String fields[] = {"x:I", "y:I"};
 		String className = "Point2D";
-		for (int valueIndex = 0; valueIndex < 200000; valueIndex++) {
+		for (int valueIndex = 0; valueIndex < 200; valueIndex++) {
```

Comment:
For this test we want to see whether the GC will scan the default value on the J9Class (see https://github.com/eclipse/openj9/blob/master/runtime/gc_structs/ValueTypesIterator.hpp#L69), we know this is tested with a big number (200000) does it still exercise that path with 200?

Reply:
This was accidentally added, the tests were simply taking too long.  Thanks for pointing it out.  Could we switch this test to call `System.gc()`?

Reply:
I just switched back the number for now.